settingup.tex

```diff
@@ -102,7 +102,7 @@ \subsection{Replacing the Real-Time Clock Battery}
 
 The battery is located between the controller ports and the keyboard connector:
 
-\includegraphics[width=10cm]{images/rtc-battery-location}
+\includegraphics[width=\linewidth]{images/illustrations/rtc-battery-location.pdf}
```

Comment:
The diagram did take up a big chunk of the page, and left that 'OPERATION' title at the bottom dangling a bit lonely:

![image](https://user-images.githubusercontent.com/20013309/139865744-35c7cb1e-31f5-41b1-b1dd-5aa256760549.png)

Could we reduce the size a little, to let some of the following page's content squeeze into here too? Or alternatively, maybe we can force a new page to assure the 'OPERATION' title moves to the next page... Let's ponder it tomorrow.

But apart from that, nice 🙂

Reply:
Probably because I used the linewidth instead of 10cm. I'll reduce it back and see how it looks.

Reply:
All good to try again, @gurcei flow is restored using a 10cm width.